charts/openshift-etcd-backup/values.yaml

```
@@ -65,6 +65,8 @@ securityContext:
   privileged: true
   # -- Set user ID
   runAsUser: 0
+  # -- Define supplemental groups
+  supplementalGroups: [""]
```

```suggestion

podSecurityContext:
  # -- Define supplemental groups
  supplementalGroups: [""]
```
do we need an empty string as default here or would an empty array (or even an empty `podSecurityContext`) make more sense?

---

you're right, an ampty `podSecurityContext` would fit best!